specification/datafactory/resource-manager/Microsoft.DataFactory/preview/2017-09-01-preview/entityTypes/Dataset.json

```diff
@@ -166,7 +166,7 @@
     },
     "JsonFormatFilePattern": {
       "description": "JSON format file pattern. A property of JsonFormat.",
-      "type": "string",
+      "type": "object",
```

Comment:
No point in changing the preview swagger as we are not releasing new SDK-s for this.

Reply:
ok